### ISSUE TITLE
Can't use Yast2 Users Module to Add Users to 389-ds...

### ISSUE THREAD

**Issue by jshand2015** [yast/yast-auth-client#104]
if i'm correct, yast-auth-client module allows yast users module to connect to 389-ds server and enable administrator to add ldap users.

right now it seems that the configuration is still set up the old way with openldap settings.  please change this so that yast users module can make use of the 389-ds server

**Comment by jshand2015** [comment]
[389ldap.txt](https://github.com/yast/yast-auth-client/files/8173169/389ldap.txt)
[ldap.txt](https://github.com/yast/yast-auth-client/files/8173170/ldap.txt)

**Comment by jshand2015** [comment]
the 389ldap.txt was copied to /etc/openldap and ldap.conf

ldap.txt is the default 389-ds configuration found in /etc folder as per 389-ds server config

the 389ldap.txt was adjusted and then placed in /etc/openldap folder so yast2 users module would sort of work.  what i mean here is that a dialogue actually brought up those settings i made in the openldap/ldap.conf file.  i entered the password and then it stopped working.  i have to exit out of ssh as it wouldn't come back with an error for ages.

**Comment by jshand2015** [comment]
![yast2-usersmodule-ldapusers](https://user-images.githubusercontent.com/10791292/156456684-5d3c0ebf-97a8-48a7-b37b-f56bc4708197.png)

**Comment by Firstyear** [comment]
I don't think this module is what manages users in yast? Also that user and group admin is for local users no? Not ldap users? 

Regardless, we'd probably remove that feature from yast as the dsidm command is better suported within the 389-ds ecosystem.

**Comment by ancorgs** [comment]
In the past, yast2-users was better integrated with LDAP and the related YaST modules. Since the rewrite of those authentication client/server modules that integration is lost. See a summary of the situation here: https://github.com/yast/yast-users/blob/master/doc/auth-modules.md

**Comment by Firstyear** [comment]
Currently there isn't really much interest to expand yast to be a full ldap/iam/idm access management tool, as it's not really in the best place to provide this functionality.
